lib/cn/cn.c

```diff
@@ -1043,7 +1043,7 @@ cn_open(
             " hb %lu%c/%lu kb %lu%c/%lu vb %lu%c/%lu %s%s%s%s%s%s",
             cn->cn_kvdb_alias, cn->cn_kvsname, (ulong)cnid,
             cn->cp->pfx_len,
-            cn->rp->value_compression, cn->rp->vcompmin,
+            cn->rp->compression.value.algorithm, cn->rp->compression.value.min_length,
```

Comment:
.value seems unncessary..

Reply:
.value is what existed before. If we are never going to compress anything other than values, then I am happy to clean it up.

Reply:
We could compress keys, but then instead of the faster KVDB on the planet we'd be the slowest :-)

Reply:
The true power is switching between the two with the flip of a switch!